src/components/navbar/styles.ts

```diff
@@ -1,7 +1,23 @@
 import { type SxProps } from "@mui/material";
+
 import { breakpoints, theme } from "~/styles/theme.js";
 
 export const styles = {
+  container: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 2,
```

Comment:
Is it worth adding some values for zIndex as constants in the design system declarations?

Reply:
In my opinion z-index is the devil and the ideal frontend need not use it. That being said yeah that's probably a good idea. I'm not familiar with the design system implementation here but I'm happy to take a deeper look, shall I add a todo?

Reply:
added todo